widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -8297,4 +8307,20 @@ body.bootstrap .modal.confirmationDialog .dialog div.body {
 
 .is-hidden {
   display: none;
+}
+
+.custom-dropdown:focus-visible,
+.custom-dropdown:hover{
+  background-color: #CC0000;
+  color: #FFFFFF;
+}
+
+.custom-dropdown:hover .custom-dropdown-label,
+.custom-dropdown:focus-visible .custom-dropdown-label{
+  color: #FFFFFF;
+}
+
+.custom-dropdown-popup .gwt-MenuBar-vertical .gwt-MenuItem.gwt-MenuItem-selected{
+  background-color: #e0e0e0;
+  color: #000000;
```

Comment:
I would ask you to first try to place these new styles instead `mantleRuby.css`, as that's where _most_ custom dropdown styles are placed, given these classes belong to a GWT component defined in mantle.

If for some reason, these need to be here, then, please place them near the existing styles (e.g. for `custom-dropdown-arrow`). AFAICT, these are placed here, and not in `mantleRuby.css`, for easy access to shared images.

Reply:
Will be creating a PR for mantleRuby.css